api_app/ingestors_manager/ingestors/virus_total.py

```diff
@@ -0,0 +1,350 @@
+import logging
+from typing import Any, Iterable
+from unittest.mock import patch
+
+from django.utils import timezone
+
+from api_app.ingestors_manager.classes import Ingestor
+from api_app.mixins import VirusTotalv3AnalyzerMixin
+from tests.mock_utils import MockUpResponse, if_mock_connections
+
+logger = logging.getLogger(__name__)
+
+
+# apply a filter to all query results, let's try to reduce the FPs
+def filter_vt_search_results(result):
+    file_to_download = []
+    data = result.get("data", {})
+    logger.info(f"Retrieved {len(data)} items from the query")
+    for d in data:
+        attributes = d.get("attributes", {})
+
+        # https://virustotal.readme.io/reference/files
+        threat_severity = attributes.get("threat_severity", {})
+        threat_severity_level = threat_severity.get("threat_severity_level", "")
+
+        if threat_severity_level in ("SEVERITY_MEDIUM", "SEVERITY_HIGH"):
+            file_to_download.append(d["id"])
+    logger.info(
+        f"Filtered {len(data) - len(file_to_download)} FP elements, "
+        f"processing {len(file_to_download)} samples"
+    )
+    return file_to_download
+
```

Comment:
this filter is based on our custom use case and decision and should not be shipped "by default" imho. People should be allowed to filter if and how they want.
Could you enable this filter based on a boolean flag like `sigma_severity_filter`?